src/services/ReadModelGateway.ts

```diff
@@ -0,0 +1,17 @@
+/*  =======================================================================  
+  IMPORTANT: This service mocks all operations performed throught read models 
+===========================================================================  */
+
+import { EService } from "../model/domain/models.js";
+
+export interface IReadModelGateway {
+  readonly getEServiceByName: (name: string) => Promise<EService | undefined>;
+  readonly getOrganizationID: () => Promise<string>;
+}
+
+export const ReadModelGateway: IReadModelGateway = {
```

Comment:
minor, ma per convenzione terrei i valori con lettera minuscola

```suggestion
export const readModelGateway: IReadModelGateway = {
```